conf/minifi-log.properties

```diff
@@ -68,3 +68,6 @@ logger.org::apache::nifi::minifi=INFO,rolling
 ## Setting any of these to 0 disables the in-memory log compression.
 #compression.cached.log.max.size=8 MB
 #compression.compressed.log.max.size=8 MB
+
+## Maximum length of a MiNiFi log entry
+#max.log.entry.length=1024
```

Comment:
I would add ", use -1 for unlimited" to the comment.

Also, I don't insist, but I think "=unlimited" would be better than "=-1".

Reply:
I would keep both options, added comment and accepting of "unlimited" value in 9133ea4134b5fc1c68c09c000211e69a0eebf2a0